app/src/main/java/org/schabi/newpipe/local/feed/item/StreamItem.kt

```diff
@@ -30,6 +31,12 @@ data class StreamItem(
     private val stream: StreamEntity = streamWithState.stream
     private val stateProgressTime: Long? = streamWithState.stateProgressMillis
 
+    /**
+     * Will be executed at the end of the [StreamItem.bind] (with (ListStreamItemBinding,Int)).
```

Comment:
```suggestion
     * Will be executed at the end of the [StreamItem.bind] (with (ListStreamItemBinding)).
```

Reply:
This is used in the method ``bind(viewBinding: ListStreamItemBinding, position: Int)``.
Why the change?

Reply:
Note: I also mentioned the overloaded method explicitly because of https://github.com/Kotlin/dokka/issues/80

Reply:
Oh ok, sorry, I thought that "(with ListStreamItemBinding,Int)" meant that `execBindEnd` takes two parameters of those rypes, but it doesn't, hence the request for change. But I misunderstood, so ignote my suggestion